bitnami/jaeger/templates/query/deployment.yaml

```diff
@@ -0,0 +1,192 @@
+            info "Take it easy. Could be authentication errors since the cassandra pod startup until the password is set"
+            if ! retry_while "check_cassandra_connection"; then
+            error "Could not connect to the database server"
+            exit 1
+            else
+            info "Connection check success"
+            sed -i 's/cqlsh -e "show version"/cqlsh -e "show version" -u $CASSANDRA_USERNAME -p $CASSANDRA_PASSWORD /' /cassandra-schema/create.sh
+            bash -x /cassandra-schema/docker.sh
+            fi
+        env:
+        - name: CQLSH
+          value: /opt/bitnami/cassandra/bin/cqlsh
+        - name: CQLSH_HOST
+          value: {{ printf "%s-cassandra" (include "common.names.fullname" .) }}
+        - name: CQLSH_PORT
+          value: {{ .Values.cassandra.containerPorts.cql | quote }}
+        - name: CASSANDRA_USERNAME
+          value: {{ .Values.cassandra.dbUser.user }}
+        - name: CASSANDRA_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ printf "%s-cassandra" (include "common.names.fullname" .) }}
+              key: cassandra-password
+        volumeMounts:
+        - name: cassandra-schema
+          mountPath: "/cassandra-schema"
+      {{- if .Values.query.initContainers }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.query.initContainers "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: jaeger-query
+          image: {{ include "jaeger.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.query.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.query.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.query.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.query.command "context" $) | nindent 12 }}
+          {{- else }}
+          command: ["/opt/bitnami/jaeger/bin/jaeger-query"]
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.query.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.query.args "context" $) | nindent 12 }}
+          {{- else }}
+          {{- end }}
+          {{- if .Values.query.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.query.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: ui
+              containerPort: {{ .Values.query.service.ports.api }}
```

Comment:
api or ui?

Reply:
Fixed within the improvements of [a896c22](https://github.com/bitnami/charts/pull/13480/commits/a896c223d6f2e265ad494dd42b6553552d0a89b0)